src/routes/voteOnSponsorTime.ts

```diff
@@ -176,6 +176,19 @@ async function categoryVote(UUID: SegmentUUID, userID: UserID, isVIP: boolean, i
         return res.status(400).send("Cannot vote for this category");
     }
 
+    // Ignore vote if the next category is locked
+    const nextCategoryLocked = await db.prepare("get", `SELECT "videoID", "category" FROM "lockCategories" WHERE "videoID" = ? AND "category" = ?`, [videoInfo.videoID, category]);
+    if (nextCategoryLocked) {
+        if (!isVIP) {
+            return res.sendStatus(200);
+        } // In an else statement, add a warning in the future for VIPs, that the next category is locked
+    }
```

Comment:
I think this can be handled client side instead, so would be better as
```suggestion
    if (nextCategoryLocked && !isVIP) {
        return res.sendStatus(200);
    }
```